README.md

```diff
@@ -0,0 +1,47 @@
+# Faraday::LogFilter
+
+A Faraday middleware for params filtering in logs.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem "faraday-log_filter", github: "westfieldlabs/faraday-log-filter"
```

Comment:
the mixture of `-` and `_` seems a little strange here, why not use all underscores and call it `faraday_log_filter`? Or is `faraday-<middleware>` the usual naming conventoin of faraday middleware?